PizzaOvenUI/Utility.qml

```diff
@@ -77,16 +77,20 @@ Item {
         var menuItems = menuSettings.json.menuItems;
 
         console.log("In findPizzaTypeFromSettings.");
+        console.log("Requested settings - dome: " + dome + ", stone: " + stone + ", time: " + cookTime);
 
+        console.log("Searching cycles for a match.");
         for(var i=0; i<menuItems.length; i++)  {
+            console.log("name: " + menuItems[i].name + ", dome: " + menuItems[i].domeTemp + ", stone: " + menuItems[i].stoneTemp + ", time: " + menuItems[i].cookTime);
             if (menuItems[i].domeTemp == dome && menuItems[i].stoneTemp == stone && cookTime == menuItems[i].cookTime) {
                 console.log("Found matching settings, setting the food index to " + i);
                 rootWindow.foodIndex = i;
                 return;
             }
         }
-        console.log("No match found, setting the food index to 4");
+        console.log("No match found, setting the food index to 4 and save these new settings as custom.");
         rootWindow.foodIndex = 4;
+        saveCurrentSettingsAsCustom();
```

Comment:
This fix allows the custom cycle to be started from the app. Whenever the custom cycle is started from the oven UI, it is always stored like this. Now they behave the same and play nicely together.

Reply:
That's good that it's acting the same as the UI. I didn't realize that we ever change that file, though. I am providing the menu settings to the cloud as a static ERD that shouldn't change. 

Basically, if we leave it the way it is, the `Custom` setting on the App will be whatever the `Custom` setting was when the oven started up. Otherwise we could to have the cloud subscribe to the 3 ERDs and publish updates when the custom setting is changed. Currently, we just just use a static value for the Gen 1 ovens, so using the value at startup is providing roughly the same level of functionality, so I think that would be ok. Thoughts?

Reply:
Are you saying that I should remove this save call? Here is the problem:
1. Modify an existing cycle in the app, let's say 515, 275, 60.
2. When we get to this function, it will not find a match but will set the `foodIndex = 4`, which is Custom.
3. When `Screen_AwaitStart` loads, it will read the temps from the json file for `foodIndex`, which will load the Custom cycle settings (1200, 675, 120 I think). 

So if we don't save the new custom settings, they will not be properly loaded.  I may be able to find another way to do this if you think that makes sense.

Reply:
I think it's good to have this change and we should definitely include it. I was just thinking through the implications of that file changing and what the user experience will be. Currently, the values what we send to the App for custom are never updated; they're hardcoded in the app. With all of our changes that are going in, we will now send the values for Custom to the cloud when the oven reconnects to the cloud. So, we won't show the most up to date value stored on the unit for Custom, but we weren't updating Custom in the app anyway, so we haven't broken anything.